orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/LogoutService.java

```diff
@@ -120,6 +120,8 @@ private void destroySessions(Session session) {
         }
         LOG.info("Deleting Session");
         sessionService.deleteStoredSession(session.getSessionId());
+        LOG.info("Deleting Orch Session");
+        orchSessionService.deleteStoredOrchSession(session.getSessionId());
```

Comment:
What's the plan when the shared session gets deprecated? The Orch session will at some point need to be passed  to `destroySessions()` to get `sessionId`, right?

Reply:
My thinking is that this was the minimum change required to get this through asap. For sure, eventually we want to add orchSession to the `destroySessions` interface, but we need that for the clientSessions and email as well. So I'll be updating the interface for email, and at that point, the change to use orchSession.getSessioId() will be a simple change.

Reply:
Sounds good to me!